### PR TITLE
Update to use piped-base:v0.1.2 image

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -133,7 +133,7 @@ container_pull(
     digest = "sha256:e357a240b1e8668c851a1dcb3e3244b4e037aab56029f394dda6925070d473a9",
```
Comment (nghialv) [inline WORKSPACE:133]: This SHA must be updated too.

Comment (nghialv) [reply WORKSPACE:133]: https://console.cloud.google.com/gcr/images/pipecd/GLOBAL/piped-base@sha256:e863054859305080e89741bad1444e46a05ecb57cf2faf33a4bd448a992c47f6/details?tab=info

Comment (nakabonne) [reply WORKSPACE:133]: Thanks :D

**Pull request merged**